terraform/openstack-modules.sample.tf

```diff
@@ -0,0 +1,99 @@
+variable subnet_cidr { default = "10.0.0.0/24" }
+variable public_key { default = "/home/you/.ssh/id_rsa.pub" }
+
+variable name { default = "mantl" }     # resources will start with "mantl-"
+variable control_count { default = "3"} # mesos masters, zk leaders, consul servers
+variable worker_count { default = "5"}  # worker nodes
+variable edge_count { default = "2"}    # load balancer nodes
+
+# Run 'nova network-list' to get these names and values
+# Floating ips are optional
+variable external_network_uuid { default = "uuid-of-your-external-network" }
+variable floating_ip_pool { default = "name-of-your-floating-ip-pool" }
```

Comment:
would it make sense to move these two defaults to a comment? That way, Terraform would complain if they weren't set. As it is, i'll try to use `uuid-of-your-external-network` as the content of the variable if not set by the user.